### PR TITLE
Cluster infra size report

### PR DESCRIPTION
**Pull request by r2dedios**
Added flags to configure script's running:
  ./exporter.sh
    -k: Specifies the KUBECONFIG var to access to the source cluster. (Default: ~/.kube/config)
    -i: Introduces manually the desired cluster ID. (Default: obtained automatically)
    -r: Configures the results config dir. (Default: ./results)
    -h: Prints script's usage

### CODE REVIEW

File: exporter/infra/exporter.sh [hunk 1]
```diff
@@ -26,15 +51,15 @@ CLUSTER_API=$($OC cluster-info | \
 )
 # Check cluster API connection
 [[ $? -ne 0 ]] && { echo "Not connected to a cluster. Exiting..."; exit; }
-CLUSTER_ID="$($OC get clusterversion -o jsonpath='{.items[].spec.clusterID}{"\n"}')"
```
Comment (dmartinol) [inline exporter/infra/exporter.sh:29]: Please always keep the original CLUSTER_ID as it is
Instead, we can rename variable associated to "-i" option to something like $USER_CLUSTER_ID and use it as the file name but not as the CLUSTER_ID value in the .env file

Comment (r2dedios) [reply exporter/infra/exporter.sh:29]: Did it

File: exporter/infra/exporter.sh [hunk 2]
```diff
@@ -26,15 +51,15 @@ CLUSTER_API=$($OC cluster-info | \
...
 
-
-## Init
-################################################################################
-echo "Initializing..."
 # Results dir creation
 [[ ! -d $RESULTS_DIR ]] && { mkdir -p $RESULTS_DIR; }
 
+# If any cluster_id was introduced by args, it will be query to the API
+if [[ $CLUSTER_ID == "" ]]; then
```
Comment (dmartinol) [inline exporter/infra/exporter.sh:59]: This if and comment can be removes

**Review by dmartinol — APPROVED** [review]
LGTM

**Pull request merged**